packages/compass-data-modeling/src/components/drawer/diagram-editor-side-panel.tsx

```diff
@@ -144,11 +144,12 @@ export default connect(
     const model = selectCurrentModelFromState(state);
 
     if (selected.type === 'collection') {
+      const isNewCollection = !selected.id;
       const doesCollectionExist = model.collections.find((collection) => {
         return collection.ns === selected.id;
       });
 
-      if (!doesCollectionExist) {
+      if (!isNewCollection && !doesCollectionExist) {
```

Comment:
Is this a leftover from the previous version where id on a new collection was optional?